ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt

```diff
@@ -30,8 +78,62 @@ hideall "--sync--"
 #6000.0 "--Reset--" sync / 00:0839:Pride of the Lioness is no longer sealed/ window 0,2000 jump 0
 
 ### Dahu
+# -p 576D:8008.4
+# -ii 576C 5B43
 8000.0 "--sync--" sync / 00:0839:The Hall of Supplication will be sealed off/ window 10000,0
 8000.0 "--Reset--" sync / 00:0839:The Hall of Supplication is no longer sealed/ window 0,2000 jump 0
+8008.4 "Reverberating Roar" sync /:Dahu:576D:/ window 10,2.5
+8013.5 "Falling Rock x6" duration 10 #sync /:Dahu:576E:/
+8014.5 "Reverberating Roar" sync /:Dahu:576D:/
+8022.7 "Hot Charge 1" sync /:Dahu:5773:/
+8027.5 "Hot Charge 2" sync /:Dahu:5773:/
+8034.3 "Firebreathe" sync /:Dahu:5774:/
+8046.5 "--sync--" sync /:Marchosias:576A:/
+8051.8 "Head Down" sync /:Marchosias:5768:/
+8057.5 "Head Down" sync /:Marchosias:5768:/
+8061.4 "Spit Flame" sync /:Dahu:5776:/
+8061.6 "Spit Flame 1" #sync /:Dahu:5777:/
+8062.7 "Spit Flame 2" #sync /:Dahu:5777:/
+8063.8 "Spit Flame 3" #sync /:Dahu:5777:/
+8065.0 "Spit Flame 4" #sync /:Dahu:5777:/
+8069.1 "Head Down" sync /:Marchosias:5768:/
+8071.1 "Right-Sided Shockwave/Left-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8073.7 "Left-Sided Shockwave/Right-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8074.8 "Head Down" sync /:Marchosias:5768:/
+8081.4 "Feral Howl" sync /:Dahu:5767:/
+8083.5 "--knockback--" sync /:Dahu:5B35:/ # TODO: confirm
+8084.9 "Hunter's Claw" sync /:Marchosias:5769:/
+8091.5 "Firebreathe x5" sync /:Dahu:576B:/ duration 8.7
+8112.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8117.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8122.5 "--sync--" sync /:Crowned Marchosias:576A:/
+8142.5 "--sync--" sync /:Marchosias:576A:/
+8142.5 "Reverberating Roar" sync /:Dahu:576D:/
+8147.6 "Falling Rock x6" duration 10 #sync /:Dahu:576E:/
+8147.7 "Head Down" sync /:Marchosias:5768:/
+8151.6 "Right-Sided Shockwave/Left-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8153.4 "Head Down" sync /:Marchosias:5768:/
+8154.1 "Left-Sided Shockwave/Right-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8159.1 "Head Down" sync /:Marchosias:5768:/
+8164.7 "Spit Flame" sync /:Dahu:5776:/
+8165.0 "Spit Flame 1" #sync /:Dahu:5777:/
+8166.1 "Spit Flame 2" #sync /:Dahu:5777:/
+8167.3 "Spit Flame 3" #sync /:Dahu:5777:/
+8168.4 "Spit Flame 4" #sync /:Dahu:5777:/
+8174.6 "Hysteric Assault" sync /:Dahu:5778:/
+8177.7 "Burn" sync /:Dahu:5463:/
+8178.1 "Hunter's Claw" sync /:Marchosias:5769:/
+8187.6 "Head Down" sync /:Marchosias:5768:/
+8188.9 "Firebreathe x5" sync /:Dahu:576B:/ duration 8.7
+8209.9 "Right-Sided Shockwave/Left-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+8212.5 "Left-Sided Shockwave/Right-Sided Shockwave" #sync /:Dahu:(5770|5771|5772|576F):/
+
+# TODO: is this a loop?
```

Comment:
It does loop, but not predictably. This hot charge can be hot charge 1 > Firebreathe,  or as you have written.

The adds do not spawn in evenly, preventing a nice loop.

Reply:
I think what happens is adds are based on only Hot Charge/Firebreathe timing. If it goes Hot Charge => Hot Charge => Firebreathe, then adds are spawning later than if they were Hot Charge => Firebreathe, but the other skills are impacted  by these timing differences. This sounds like a bug in the encounter to me or I don't have enough samples to show how adds spawn.

This fight is likely to be cleared before or around the 3 minute mark, so it is unlikely we have a group that is going to realistically see passed two loops.

Reply:
Is the first one always a double hot charge?

Reply:
> 
> 
> Is the first one always a double hot charge? Do you have an example log with a single hot charge?

First one is always double hot charge.

Second one can be either or:
Single - https://www.fflogs.com/reports/LZPRYhkt3NBXW4aF#fight=15&type=casts&view=events&hostility=1&eventstart=15258535
Double - https://www.fflogs.com/reports/LZPRYhkt3NBXW4aF#fight=10&type=casts&view=events&hostility=1&eventstart=12503659

Reply:
Ok, updated to handle that.  There's not much after that single, but the next few syncs including adds line up after that perfectly.

```
6221.646: Matched entry: 6221.4 Hot Charge 1 (-0.246s)
6223.271: Matched entry: 6228.0 --sync-- (+4.729s)
    Missed sync: Hot Charge 2? at 6226.2 (last seen at 6221.646)
6232.988: Matched entry: 6233.0 Firebreathe (+0.012s)
6246.281: Matched entry: 6246.3 --sync-- (+0.019s)
6251.332: Matched entry: 6251.3 --sync-- (-0.032s)
6252.192: Matched entry: 6252.2 Reverberating Roar (+0.008s)
```